config/routes.rb

```diff
@@ -0,0 +1,102 @@
+Rails.application.routes.draw do
```

Comment:
We don't have a root route specified in `config/routes.rb`. We should always specify a root route, otherwise when people visit your site by the plain domain name (e.g. `http://google.com/`) they will get a generic message from Rails that doesn't link them to anywhere else on the site.